src/components/icon-button/icon-button.tsx

```diff
@@ -0,0 +1,22 @@
+import clsx from 'clsx';
+
+import { Icon, IIconProps } from '../icon';
+
+ interface IconButtonProps extends IIconProps {
+  /**
+     * Optional send custom classes
+     */
+   classNames?: string,
```

Comment:
bitte singular verwenden `className`

Reply:
ok, bei uns in mein Arbeit verwenden wir plural und singular ist von system selber, aber ok, angepasst